Shared/Samples/Find closest facility from point/FindClosestFacilityFromPointView.swift

```diff
@@ -0,0 +1,235 @@
+            async let parameters = try closestFacilityTask.makeDefaultParameters()
+            
+            // Get the feature layers on the map.
+            await map.operationalLayers.load()
+            let facilitiesLayer = map.operationalLayers.first(
+                where: { $0.name == "sandiegofacilities" }
+            ) as? FeatureLayer
+            let incidentsLayer = map.operationalLayers.first(
+                where: { $0.name == "sandiegoincidents" }
+            ) as? FeatureLayer
+            
+            // Get the feature tables from the feature layers.
+            guard let facilitiesTable = facilitiesLayer?.featureTable as? ArcGISFeatureTable,
+                  let incidentsTable = incidentsLayer?.featureTable as? ArcGISFeatureTable
+            else { return }
+            
+            // Create query parameters that will return all the features.
+            let queryParameters = QueryParameters()
+            queryParameters.whereClause = "1=1"
+            
+            // Set the parameters' facilities and incidents using the tables.
+            try await parameters.setFacilities(
+                fromFeaturesIn: facilitiesTable,
+                queryParameters: queryParameters
+            )
+            try await parameters.setIncidents(
+                fromFeaturesIn: incidentsTable,
+                queryParameters: queryParameters
+            )
+            closestFacilityParameters = try await parameters
+        }
+        
+        /// Finds the closest facility routes for the incidents.
+        func solveRoutes() async throws {
+            guard let closestFacilityParameters else { return }
+            
+            // Get the closest facility result from the task using the parameters.
+            let closestFacilityResult = try await closestFacilityTask.solveClosestFacility(
+                using: closestFacilityParameters
+            )
+            
+            // Create a route graphic for each incident in the result.
+            let incidentsIndices = closestFacilityResult.incidents.indices
+            let routeGraphics = incidentsIndices.compactMap { incidentIndex -> Graphic? in
+                // Get the index for the facility closest to the given incident.
+                guard let closestFacilityIndex = closestFacilityResult.rankedIndexesOfFacilities(
+                    forIncidentAtIndex: incidentIndex
+                ).first else { return nil }
+                
+                // Get the route for the facility and incident.
+                let closestFacilityRoute = closestFacilityResult.route(
+                    toFacilityAtIndex: closestFacilityIndex,
+                    fromIncidentAtIndex: incidentIndex
+                )
+                
+                // Create a graphic using the route's geometry.
+                return Graphic(geometry: closestFacilityRoute?.routeGeometry, symbol: routeSymbol)
+            }
```

Comment:
Good to move closestFacilityRoute as part of guard as no point in creating the `Graphic` if route geometry is not available.
```suggestion
            let routeGraphics = incidentsIndices.compactMap { incidentIndex -> Graphic? in
                // Get the index for the facility closest to the given incident and facility route.
                guard let closestFacilityIndex = closestFacilityResult.rankedIndexesOfFacilities(
                    forIncidentAtIndex: incidentIndex
                ).first,
                      let closestFacilityRoute = closestFacilityResult.route(
                        toFacilityAtIndex: closestFacilityIndex,
                        fromIncidentAtIndex: incidentIndex
                      ) else {
                    return nil
                }
                
                // Create a graphic using the route's geometry.
                return Graphic(geometry: closestFacilityRoute.routeGeometry, symbol: routeSymbol)
            }
```